### ISSUE TITLE
Exporting using Craft's Export feature yields blank address data

### ISSUE THREAD

**Issue by chadwells** [doublesecretagency/craft-googlemaps#91]
When exporting entries with Address fields, the "Addresses (expanded)" option will output blank data for Address fields/subfields.

Using "Addresses (condensed)" will yield the addresses in the single field as expected.

Using:
Craft 4.4.15
Google Maps 4.3.5

Issue exists on multiple sites of ours.

Truncated example of "Addresses (expanded)" export to CSV:
```
ID,Title,"Dealer Address - Street Address","Dealer Address - Apartment or Suite","Dealer Address - City","Dealer Address - State","Dealer Address - Zip Code","Dealer Address - Country","Dealer Address - Name","Dealer Address - County or District","Dealer Address - Place ID","Dealer Address - Latitude","Dealer Address - Longitude","Dealer Address - Zoom"
1184520,Audiotehnika.lv,,,,,,,,,,56.9582149,24.1100205,19
1184498,LydProdukter,,,,,,,,,,55.69751,12.581713,12
1173264,"Music To My Ear",,,,,,,,,,40.5142839,"	-80.0071567",10
```

Same export but as "Addresses (condensed) to CSV:
```
ID,Title,"Dealer Address"
1184520,Audiotehnika.lv,"Elizabetes iela 31, Centra rajons, Rīga, LV-1010, Lettonie"
1184498,LydProdukter,"Willemoesgade 16, 2100 København, Danemark"
1173264,"Music To My Ear","3003 Babcock Blvd, Pittsburgh, PA 15237, USA"
```

**Comment by lindseydiloreto** [comment]
Hi @chadwells, thanks for reporting!

I think we've got this fixed. Would you mind testing the latest dev version when you get a chance?

```json
"doublesecretagency/craft-googlemaps": "dev-v4-dev"
```

**Comment by lindseydiloreto** [comment]
Good news, this fix has been officially released in [v4.3.6](https://github.com/doublesecretagency/craft-googlemaps/releases/tag/4.3.6). 👍
